include/umf/memspace.h

```diff
@@ -61,6 +61,11 @@ umf_memspace_handle_t umfMemspaceHighestCapacityGet(void);
 ///
 umf_memspace_handle_t umfMemspaceHighestBandwidthGet(void);
 
+/// \brief Retrieves predefined lowest latency memspace.
+/// \return lowest latency memspace handle on success or NULL on failure.
+///
+umf_memspace_handle_t umfMemspaceLowestLatencyGet(void);
```

Comment:
maybe it is worth to add a note why we could fail - e.g. "NULL on failure (no HMAT support)" here and in other Memspace*Get() functions?

Reply:
Sure, done.